todo-mvc-tsx/README.md

```diff
@@ -0,0 +1,76 @@
+# Dojo2 TodoMVC Example
+
+A reference implementation of [TodoMVC](http://todomvc.com/) built with current pre-release versions of Dojo 2 packages.
+
+- [Pre-requisites](#pre-requisites)
+- [Usage](#usage)
+- [Features](#features)
+- [How do I contribute?](#how-do-i-contribute)
+  - [Installation](#installation)
+  - [Testing](#testing)
+- [Licensing information](#licensing-information)
+
+## Pre-requisites
+
+`@dojo/cli` must be installed globally:
+
+```shell
+npm install @dojo/cli -g
+```
+
+## Usage
+
+To use `todo-mvc`, clone the `examples` repository and install the npm dependencies:
+
+```shell
+git clone https://github.com/dojo/examples
+cd examples/todo-mvc
+npm install
+```
+
+To run the application simply run the dojo build.
+
+```
+dojo build
+```
+
+To watch and serve the application source, then run the build with the `-w` command.
+
+```
+dojo build -w
+```
+
+## Features
+
+This package demonstrates how to build a simple application using Dojo 2 widgets.
```

Comment:
"using Dojo 2 widgets with the alternative tsx syntax, a TypeScript version of jsx." or something like that? Otherwise it sounds the same as the normal todo app